src/pages/index.js

```diff
@@ -1,6 +1,6 @@
 import dynamic from 'next/dynamic';
 
-const DynamicApp = dynamic(() => import('../App'), { ssr: false });
+const DynamicApp = dynamic(() => import('../Home'), { ssr: true });
```

Comment:
When I change ssr to true, I got the window object not found error, just like what we have discussed, right? 
 For the Home page, we want to render the content on the left on server-side, and render the map on the right on client-side, so we need to resolve the window-importing problem?
Any idea on this?

```
A codemod is available to fix the most common cases: https://nextjs.link/codemod-ndc
info  - ready on http://localhost:3000
ReferenceError: window is not defined
    at /Users/deanchen/work/greenstand/treetracker-web-map-client/node_modules/leaflet/dist/leaflet-src.js:227:1
9
    at /Users/deanchen/work/greenstand/treetracker-web-map-client/node_modules/leaflet/dist/leaflet-src.js:7:66
    at Object.<anonymous> (/Users/deanchen/work/greenstand/treetracker-web-map-client/node_modules/leaflet/dist/
leaflet-src.js:10:2)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.leaflet (/Users/deanchen/work/greenstand/treetracker-web-map-client/.next/server/pages/index.js:40
7:18)
    at __webpack_require__ (/Users/deanchen/work/greenstand/treetracker-web-map-client/.next/server/webpack-runt
ime.js:33:42)
    at eval (webpack-internal:///./src/App.js:2:65)
    at Object../src/App.js (/Users/deanchen/work/greenstand/treetracker-web-map-client/.next/server/src_Home_js.
js:40:1)
    at __webpack_require__ (/Users/deanchen/work/greenstand/treetracker-web-map-client/.next/server/webpack-runt
ime.js:33:42)
    at eval (webpack-internal:///./src/Home.js:5:62)
    at Object../src/Home.js (/Users/deanchen/work/greenstand/treetracker-web-map-client/.next/server/src_Home_js
.js:51:1)
wait  - compiling...
warn  - ./src/components/Loader.js
Anonymous function declarations cause Fast Refresh to not preserve local component state.
Please add a name to your function, for example:
```